concepticondata/conceptlists/RonaTas-2011-431.tsv

```diff
@@ -0,0 +1,432 @@
+RonaTas-2011-431-377	377	to wind something round, to twist	430	TWIST
+RonaTas-2011-431-378	378	trough, wash tub, hutch	205	TROUGH
+RonaTas-2011-431-379	379	strap (on a whip, or on a kind of sandal)	2973	STRAP
+RonaTas-2011-431-380	380	a piece of land, field, parcel, patch, a higher place where animals rest and accumulate dung	212	FIELD
+RonaTas-2011-431-381	381	to vegetate, to linger in misery		
+RonaTas-2011-431-382	382	axle	1769	AXLE
+RonaTas-2011-431-383	383	sea	1474	SEA
+RonaTas-2011-431-384	384	to turn, to change the original direction	1588	TURN
+RonaTas-2011-431-385	385	knee	1371	KNEE
+RonaTas-2011-431-386	386	hall, chamber, great room		
+RonaTas-2011-431-387	387	camel	1871	CAMEL
+RonaTas-2011-431-388	388	hemp breaker, swingle, scutch		
+RonaTas-2011-431-389	389	steer, young bullock, ox	1169	OX
+RonaTas-2011-431-390	390	to lay eggs	2464	LAY EGGS
+RonaTas-2011-431-391	391	sturgeon, Acipenser sturio	3606	STURGEON
+RonaTas-2011-431-392	392	one-year-old lamb		
+RonaTas-2011-431-393	393	interpreter	3744	INTERPRETER
+RonaTas-2011-431-394	394	horseradish, Armoracia rusticana		
+RonaTas-2011-431-395	395	to pile up, to become blocked	1576	PILE UP
+RonaTas-2011-431-396	396	a kind of small falcon, Falco columbarius aescalon		
+RonaTas-2011-431-397	397	Slovak		
+RonaTas-2011-431-398	398	perfect		
+RonaTas-2011-431-399	399	innumerable		
+RonaTas-2011-431-400	400	to break (also e.g. hemp, salt, pepper), to separate into pieces, to break,to crush	1520	CRUSH
+RonaTas-2011-431-401	401	snare, trap	1075	TRAP (PITFALL)
+RonaTas-2011-431-402	402	Turk, Turkish, Turkic, Ottoman		
+RonaTas-2011-431-403	403	to happen, to occur	248	HAPPEN
+RonaTas-2011-431-404	404	law	1062	LAW
+RonaTas-2011-431-405	405	peat, turf, dried dung		
+RonaTas-2011-431-406	406	steer, young bullock, ox or cow	1007	COW
+RonaTas-2011-431-407	407	cottage cheese, cheese curds		
+RonaTas-2011-431-408	408	the totem bird of the Árpád dynasty, a bird of prey		
+RonaTas-2011-431-409	409	bustard, Otis		
+RonaTas-2011-431-410	410	mirror	1171	MIRROR
+RonaTas-2011-431-411	411	to endure, suffer, bear, stand	833	ENDURE
+RonaTas-2011-431-412	412	to roll up (a scroll or ones sleeves)		
+RonaTas-2011-431-413	413	hen	1514	HEN
+RonaTas-2011-431-414	414	owl, Strix	735	OWL
+RonaTas-2011-431-415	415	gentleman, lord, sir, high dignity, Lord (also of a king or of God)	3153	LORD
+RonaTas-2011-431-416	416	to refresh oneself, to rest, to become cured	168	REST
+RonaTas-2011-431-417	417	feast, ünnepnap		
+RonaTas-2011-431-418	418	heifer, roe doe		
+RonaTas-2011-431-419	419	ground squirrel, Citellus citellus		
+RonaTas-2011-431-420	420	wormwood, Artemisia; bitterness	887	BITTER
+RonaTas-2011-431-421	421	wether, sheep	1331	SHEEP
+RonaTas-2011-431-422	422	hot embers, extinguished embers		
+RonaTas-2011-431-423	423	one-year-old female sheep		
+RonaTas-2011-431-424	424	to hollow out, to scoop out	46	HOLLOW OUT
+RonaTas-2011-431-425	425	trough, tray	205	TROUGH
```

Comment:
This entry refers to `2629 TRAY`, so I'd change the mapping

Reply:
We have two concepts for each of the glosses, so I'd unmap.

Reply:
Are you sure? As I understand it, 'through' ("Trog") is meant, not ' through' ("durch"). Then this entry corresponds to only one concept.

Reply:
We have 205 TROUGH (Trog) and 2629 TRAY as concept sets in Concepticon. See:
https://concepticon.clld.org/parameters/205 and https://concepticon.clld.org/parameters/2629

Reply:
aaay now I understand, thank you! (:

Reply:
Resolved in 70cb680e6327a35615c7b0841c50828f1bee1534